### ISSUE TITLE
geoRadius - How to?

### ISSUE THREAD

**Issue by asindl** [redis/node-redis#2670]
### Description

I am running a 6.0.14 redis server (in Azure) and want to use the geoRadius search that includes the coordinates (WITHCOORD).
It is really hard to find in the documentation how exactly the call would look like.

Any idea anyone?

**Comment by leibale** [comment]
`client.GEORADIUS_WITHCOORDS`/`client.geoRadiusWithCoords` should do the trick

**Comment by asindl** [comment]
Not sure, what that means. There is no `geoRadiusWithCoords`, at least to what I see....

**Comment by asindl** [comment]
I suspect, those additional attributes are not implemented in the latest version.
I checked the file @redis/client/dist/lib/commands/generic-transformers.js and updated the function `pushGeoRadsArguments`as follows
```
function pushGeoRadiusArguments(args, key, from, radius, unit, options) {
    var geoRadiusAdditionInfo = ["WITHDIST", "WITHCOORD", "WITHHASH"];
    args.push(key);
    if (typeof from === 'string') {
        args.push(from);
    }
    else {
        args.push(from.longitude.toString(), from.latitude.toString());
    }
    args.push(radius.toString(), unit);
    geoRadiusAdditionInfo.forEach(info => {
        if (options && options[info] && options[info] == true){
            args.push(info);
        }
    });
    if (options?.SORT) {
        args.push(options.SORT);
    }
    pushGeoCountArgument(args, options?.COUNT);
    return args;
}
```
Could that be included in the lib?

**Comment by leibale** [comment]
@asindl my bad, it's not `client.GEORADIUS_WITHCOORDS` it's `client.GEORADIUS_WITH`:
```javascript
import { GeoReplyWith } from 'redis';

await client.GEORADIUS_WITHCOORDS(
  'key',
  { longitude: location[0], latitude: location[1] },
  radius,
  'km',
  [GeoReplyWith.WITH_COORDS]
);
```

in general, when a command argument changes the reply type, its implemented as its own command (to allow for better types).

**Comment by asindl** [comment]
Thanks a lot, @leibale!
Small correction: the attribute name is `COORDINATES': `GeoReplyWith["COORDINATES"] = "WITHCOORD";`

So, you example needs to be
```
import { GeoReplyWith } from 'redis';

await client.GEORADIUS_WITH(
  'key',
  { longitude: location[0], latitude: location[1] },
  radius,
  'km',
  [GeoReplyWith.COORDINATES]
);
```

Now, it works like a charm!

Since I suspected, it would work "somehow", I labeled this as "documentation". Is there any documentation about the usage of this including the response types? If not, it would be helpful to add it...

Thanks again!
